src/components/Avatar/Avatar.styles.tsx

```
@@ -0,0 +1,17 @@
+import { AVATAR_SIZE_MAP } from './Avatar.constants';
+import { AvatarProps } from './Avatar.types';
+
+/**
+ * StyledAvatar component.
+ *
+ * This component is a styled version of the MUI Avatar component
+ * with added height and width properties.
+ */
+export const StyledAvatar = styled(Avatar)<AvatarProps>(
+    ({ theme: { spacing }, size }) => ({
+        height: spacing(AVATAR_SIZE_MAP[size!]),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)